docs/HowTo/Interact/Filters/Accessing-Logs-Using-JSON-RPC.md

```diff
@@ -150,10 +154,24 @@ When a filter is no longer required, use
 [`eth_uninstallFilter`](../../../Reference/API-Methods.md#eth_uninstallfilter) to remove the
 filter.
 
+## Filters for private contracts 
+
+Filters for private contracts are created, accessed, and uninstalled using: 
+
+* [`priv_getFilterChanges`](../../../Reference/API-Methods.md#priv_getfilterchanges)
+* [`priv_getFilterLogs`](../../../Reference/API-Methods.md#priv_getfilterlogs)
+* [`priv_getLogs`](../../../Reference/API-Methods.md#priv_getlogs)
+* [`priv_newFilter`](../../../Reference/API-Methods.md#priv_newfilter)
+* [`priv_uninstallFilter`](../../../Reference/API-Methods.md#priv_uninstallfilter).  
+  
+The [privacy group ID](../../../Concepts/Privacy/Privacy-Overview.md) must be specified as a parameter
```

Comment:
Should we clarify that the privacyGroupId is the parameter index 0 in the request or will we have a request example somewhere in the docs?

Reply:
If you click through to the reference material you'll see examples but it's probably a good idea to include one here as well so I'll add that.